### DISCUSSION TITLE
[Side Entrance] -  EVM ERROR REVERT

### DISCUSSION THREAD

**Discussion by JJScar** [tinchoabbate/damn-vulnerable-defi#73] [Help / Q&A / Troubleshooting]
Hello all!

Just to play around with the contract I started calling all the functions, but it seems that EVM throws when I call `flashloan()`. 
Does anyone know what I'm doing wrong?

```
SideEntranceChallenge::test_sideEntrance()
    ├─ [0] VM::startPrank(player: [0x44E97aF4418b7a17AABD8090bEA0A471a366305C], player: [0x44E97aF4418b7a17AABD8090bEA0A471a366305C])
    │   └─ ← [Return] 
    ├─ [2970] SideEntranceLenderPool::flashLoan(1000000000000000000000 [1e21])
    │   └─ ← [Revert] EvmError: Revert
    └─ ← [Revert] EvmError: Revert
```

**Comment by tinchoabbate** [comment]
Hey @JJScar, have you made any progress in the challenge? Were you able to solve this problem you ran into?

**Comment by nidhi2026** [comment]
@JJScar, I think the problem might be directly calling flashLoan as player. We need to call flashLoan with a contract having `IFlashLoanEtherReceiver` interface (containing execute method), so directly calling as player will result in that error
